test/response/invalid.md

```diff
@@ -148,4 +148,54 @@ off=8 version complete
 off=13 len=2 span[status]="OK"
 off=17 status complete
 off=18 error code=30 reason="Unexpected space after start line"
+```
+
+### Extra space between HTTP version and status code
+
+<!-- meta={"type": "response-only"} -->
+```http
+HTTP/1.1  200 OK
+
+
+```
+
+```log
+off=0 message begin
+off=5 len=3 span[version]="1.1"
+off=8 version complete
+off=9 error code=13 reason="Invalid status code"
+```
+
+### Extra space between status code and reason
```

Comment:
Shouldn't this case be rejected as well?

Reply:
I think it should be accepted, because the spec does not say that the reason phrase may not start with a space:
`reason-phrase  = *( HTAB / SP / VCHAR / obs-text )`

Reply:
I see. Shall we want to be strict and enforce it for good practice?

Reply:
That would be stricter than what's in the spec though, it doesn't seem like it's helpful?